tools/dependencies/checkLicense.sh

```diff
@@ -42,4 +40,4 @@ echo '=== Third party dependencies: ' && grep -vf self-modules.txt all-dependenc
 # command in target OS is different from what we used to sort the file `known-dependencies.txt`, i.e. "sort the two file
 # using the same command (and default arguments)"
 
-diff -w -B -U0 <(sort < tools/dependencies/known-dependencies.txt) <(sort < third-party-dependencies.txt)
+echo '=== diff dependencies: ' && diff -w -B -U0 <(sort < tools/dependencies/known-dependencies.txt) <(sort < third-party-dependencies.txt)
```

Comment:
hi, I disabled this in #2613 so we won't be going to execute this script

Reply:
@CalvinKirs PTAL.